slack_bolt/context/ack/internals.py

```diff
@@ -1,42 +1,72 @@
-from typing import Optional, List, Union, Any
+from typing import Optional, List, Union, Any, Dict
 
 from slack_sdk.models.attachments import Attachment
 from slack_sdk.models.blocks import Block, Option, OptionGroup
+from slack_sdk.models.views import View
 
 from slack_bolt.error import BoltError
 from slack_bolt.response import BoltResponse
-from slack_bolt.util.utils import convert_to_dict_list
+from slack_bolt.util.utils import convert_to_dict_list, _to_dict
```

Comment:
Why is `_to_dict` prepended with `_` yet passed around for use other than where it's defined?

Reply:
We should rename it to `convert_to_dict`. I will do it.